ts/utils/format_number.ts

```diff
@@ -366,3 +367,22 @@ export function formatZrx(num: NumStrBigNumber, opts: FormattedCryptoOptions = {
         ...restOpts,
     });
 }
+
+export function formatDai(num: NumStrBigNumber, opts: FormattedCryptoOptions = {}): FormattedNumber {
+    const { fromBaseUnits, ...restOpts } = opts;
+    const normalizedNum: NumStrBigNumber = fromBaseUnits
+        ? Web3Wrapper.toUnitAmount(createBigNumber(num), constants.DECIMAL_PLACES_ZRX)
```

Comment:
NIT: I'd prefer to have a new constant for `DECIMAL_PLACES_DAI` it has the same number as `ZRX` feel clearer that way

Reply:
Definitely a mistake! Thanks for the callout :) Fixed